docs/DESIGN.md

```diff
@@ -10,29 +10,29 @@ liquidity providers (LPs) that store their BTC in RSK and Bitcoin. The first ver
 peg-in process (BTC to RBTC). Later versions will also support the peg-out process (RBTC to BTC).
 
 
-# Workflow:
+## Workflow
 
 The following diagram shows the interaction of the different components during the peg-in process. Note that the call
 to isOperational could be performed at a different time:
 
-![Flyover Sequence Diagram](diagrams/flyover-sd.png)
+![Flyover Sequence Diagram](https://raw.githubusercontent.com/rsksmart/liquidity-provider-server/refs/heads/flyover-2.3.0/docs/diagrams/flyover-sd.png)
```

Comment:
Question: does the dev portal support mermaid? because we could point to that file instead of the image

Reply:
also this is pointing to the 2.3.0 branch